common/STcommon.py

```diff
@@ -1,10 +1,45 @@
 try:
 	import logging
+	import argparse
+	import configparser
 	from Crypto.PublicKey import RSA
 except ImportError as e:
 	print("[-] {}, exiting".format(e))
 	exit(1)
 
+class Settings():
+	settings = configparser.ConfigParser()
+	settingsFile = ""
+	loaded = False
+
+	def createSettings(self,settingsFile='STcommon/settings.ini'):
+		self.settingsFile = settingsFile
+		with open(self.settingsFile,'w') as outFile:
+			self.settings.write(outFile)
+		self.loaded = True
+		return True
+
+	def loadSettings(self,settingsFile='STcommon/settings.ini'):
+		self.settings.read(settingsFile)
+		if len(self.settings.sections()) == 0:
+			return False
+		else
+			self.loaded = True
+			self.settingsFile = settingsFile
+			return True
+
+	def writeSetting(self,key,value,section='DEFAULT'):
+		if !self.loaded:
```

Comment:
github is marking this `!` in red which is weird. lets change it to `not` to get rid of the red and make it slightly more readable to new people.